### ISSUE TITLE
Panel issues in develop branch

### ISSUE THREAD

**Issue by kmscode** [mRemoteNG/mRemoteNG#802]
Current develop branch

- [x] New Connection "Panel" is not populated with "General" by default
- [x] Quick Connections are prompted for a Panel
- [x] If no panel is open, clicking "new" on the "select panel" dialog just closes the dialog (cancel opens the connection on "New Panel"
- [x] Right Click on Panel Tab -> Rename -> Nothing happens

**Comment by sparerd** [comment]
For item 2, is your Tabs & Panels -> "Always show panel selection dialog when opening connections" enabled? I wasn't able to replicate that one

**Comment by june07** [comment]
I'm seeing the same error as kmscode on #2.  Tried enabling/disabling  "Always show panel selection dialog when opening connections" but to no avail.

**Comment by sparerd** [comment]
The other potential cause for item 2 is that the default connection value for "Panel" is empty. Quick connections take most of their settings from the default connectioninfo. If a connection's "Panel" setting is empty, the app opens the choose panel form. If that isn't desirable, we can make it an option to either prompt or just use the General panel.

**Comment by sparerd** [comment]
Not able to replicate items 3 and 4 in v1.76 alpha 2 either. @kmscode can you confirm? It might be something specific to your instance, so I'd like to be certain i'm not missing something.

Come to think of it, item 1 might be related to my last post. It's likely your default connection "Panel" property is empty

**Comment by kmscode** [comment]
will do!

**Comment by kmscode** [comment]
Looks like items 3 & 4 no longer occur.

Item 1 (panel for new connection is not populated): 
![image](https://user-images.githubusercontent.com/16654335/43209282-458c40c4-8ffa-11e8-8adb-cbe4bcd4fa5b.png)

Item 2 (panel prompt for quick connection):
![image](https://user-images.githubusercontent.com/16654335/43209343-5dcf151c-8ffa-11e8-9926-e4b35786b8c6.png)


This is called from the constructor in ConnectionInfo.cs - so it should be set properly:
```
        private void SetTreeDisplayDefaults()
        {
            Name = Language.strNewConnection;
            Description = Settings.Default.ConDefaultDescription;
            Icon = Settings.Default.ConDefaultIcon;
            Panel = Language.strGeneral;
        }
```

Not sure why it's empty - didn't have a chance to dig any further yet...

**Comment by kmscode** [comment]
Value appears to be removed in DefaultConnectionInfo.Instance.SaveTo which is called from ConnectionTree.AddNode after the connectionInfo is properly created.

I'm guessing that 52ad4435ac7177af929ade87ab6c5629c1ba32b0 caused this problem since "Panel" was removed as an excluded property.

That seems to have been moved into ConnectionInfo.cs in 35582a5e6aee078d3920e5a5c702de79a1df35ec

@sparerd - mind taking a quick look since you did this fix... I don't want to just revert the change and lose the fix.

**Comment by sparerd** [comment]
Yea, we may just need to set a default value for `Panel` in the default connection info if no value is found. Is it safe to do that? Is there ever a case where someone would *not* want a value in the `Panel` property at all?

**Comment by kmscode** [comment]
We _DO_ default it to "General" but ```DefaultConnectionInfo.Instance.SaveTo``` removes that default.

**Comment by sparerd** [comment]
Ah, I see. I'll look into fixing that today.

**Comment by sparerd** [comment]
Ok, I think this is fixed on the v1.76 branch now. 2 changes:
- Set the ConDefaultPanel setting to have a default of General (unfortunately cant use a localized string here)
- Updated the methods that create quick connect and special web connections (Help menu items) to use the localized General form if the default connectioninfo doesn't have a value set.

**Comment by kmscode** [comment]
Confirmed. Fixed in commits:
e31088fd2ea13d9ef55397d9c9b09ee9259870d1
1d7bb6371090d7eb0e413cd1d3d6cdcb8cd2b6d9
